### DISCUSSION TITLE
How do I debug email sending with docker set-up?

### DISCUSSION THREAD

**Discussion by michnovka** [lukevella/rallly#207] [General]
I am not receiving any e-mails. I have configured .env file, restarted with docker-compose but still cant get the emails to be sent. I tried to ping my mail server and test SMTP connection from inside the container using `nc` and `ping` and there are no connectivity issues. Is there some log of whats going on? Thanks!

**Comment by assaner2020** [comment]
same problem here. I have included this on the `.env` file:
`SMTP_SECURE=false`

However, I can see on my internal mail server logs that it is trying to use STARTTLS which of course fails

`Jul  8 12:19:13 a86ac30e434b postfix/smtpd[38491]: connect from unknown[my-ip-address]
Jul  8 12:19:13 a86ac30e434b postfix/smtpd[38491]: SSL_accept error from unknown[my-ip-address]: lost connection
Jul  8 12:19:13 a86ac30e434b postfix/smtpd[38491]: lost connection after STARTTLS from unknown[my-ip-address]`
